docs/Tutorials/DomainCreation.md

```diff
@@ -0,0 +1,217 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Domain Creation {#tutorial_domain_creation}
+
+### Domain Decomposition
+The spatial domain on which the solution is computed is divided into
+non-overlapping cells. For 2D and 3D domains, we use curvilinear
+quadrilateral and curvilinear hexahedral elements, respectively. The
+specification of a particular spatial domain is given by the DomainCreator.
+DomainCreators specify the properties of the initial elements (before AMR takes
+place), which are then used by Domain to create Blocks. Each Block
+holds a CoordinateMap (described below) as well as information about its
+neighbors. For geometrical domains, (e.g. rectilinear, spherical) there are a
+few shortcuts that can be used such that a user does not need to specify by
+hand the map and neighbor information for each %Block; these methods are
+explained below.
+
+## CoordinateMaps
+Each Block in the Domain must hold a CoordinateMap which describes how to
+map the logical cube (a domain that goes [-1. 1] in each dimension) to the
+curvilinear hexahedral element the %Block describes. The %CoordinateMap
+also provides the jacobian of the mapping.
+
+### Shortcuts for CoordinateMaps
+For spherical domains, Wedge3D implements the cubed-sphere map, and there
+exists the method `wedge_coordinate_maps` in DomainHelpers to quickly
+construct six of these maps at once. For rectilinear multicube domains,
+Affine and Equiangular map the logical cubes to cartesian cubes. The method
+`maps_for_rectilinear_domains` in DomainHelpers allows the user to obtain
+the CoordinateMaps for all of the %Blocks in such a domain at once. These
+can both be used to provide the map arguments to the Domain constructor.
+
+## Boundary Information
+Each Block must know which of the other Blocks in the Domain are its
+neighbors and which of its logical directions points to an external boundary.
+
+### Shortcuts for Boundary Information
+A quick way to encode the neighbor and boundary information for the blocks
+in a domain is through our convention of numbering and ordering the corners
+of the blocks in a well-defined way. The corner numbering scheme is described
+in the \ref tutorial_orientations tutorial. For spherical domains,
+DomainHelpers has the methods `corners_for_radially_layered_domains` and
+`corners_for_biradially_layered_domains`, which provides the proper corner
+numberings for the maps obtained from the method `wedge_coordinate_maps`.
+These methods are used in the Shell and Sphere DomainCreators.
```

Comment:
Maybe link Shell and Sphere (Doxygen has the `\ref DomainCreators::Shell "Shell"` command)